### PR TITLE
CONSOLE-3367: Back end code for passing up hubConsoleURL to the GUI

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://issues.redhat.com/browse/CONSOLE-3367

Proof of work:

In a multi cluster env, I ran bridge as follows: /bin/bridge -hub-console-url="http://www.google.com". To make sure this was passed correctly, I rendered the hubConsoleURL windows.SERVER_FLAG in the cluster switcher as shown in the screenshot:
Screenshot 2022-11-23 at 2 09 39 PM

### CODE REVIEW

File: cmd/bridge/main.go [hunk 1]
```diff
@@ -148,6 +148,7 @@ func main() {
 	fReleaseVersion := fs.String("release-version", "", "Defines the release version of the cluster")
 	fNodeArchitectures := fs.String("node-architectures", "", "List of node architectures. Example --node-architecture=amd64,arm64")
 	fCopiedCSVsDisabled := fs.Bool("copied-csvs-disabled", false, "Flag to indicate if OLM copied CSVs are disabled.")
+	fHubConsoleURL := fs.String("hub-console-url", "", "URL of the hub cluster in a multi cluster environment.")
```
Comment (jhadvig) [inline cmd/bridge/main.go:151]: nit:
```suggestion
	fHubConsoleURL := fs.String("hub-console-url", "", "URL of the hub cluster's console in a multi cluster environment.")
```

**Review by jhadvig — COMMENTED** [review]
@jcaianirh the change looks good to me, one small nit, otherwise LGTM 👍

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

Console Approver:
/assign @jhadvig

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve
/retest

**Comment by yapei** [comment]
confirmed that `window.SERVER_FLAGS.hubConsoleURL` value is set to the value of `-hub-console-url` value in bridge command line
/label qe-approved

**Pull request merged**